9.5/Dockerfile.rhel7

```diff
@@ -70,6 +72,12 @@ ENV BASH_ENV=${CONTAINER_SCRIPTS_PATH}/scl_enable \
 
 VOLUME ["/var/lib/pgsql/data"]
 
+# {APP_DATA} needs to be accessed by postgres user while s2i assembling
+# postgres user changes permissiond of files in APP_DATA during assemblng
```

Comment:
What about:
Being also in supplemental group `0`, postgres user can by default change the group
ownership of files in $APP_DATA to `postgres:root` (by fix-permissions script in assemble
script).  This group ownership change is needed because the s2i-built image is run under
arbitrary user uid (but gid=0).